package.json

```diff
@@ -42,6 +42,29 @@
     ],
     "main": "./out/extension",
     "contributes": {
+        "languages": [
+            {
+                "id": "yaml",
+                "aliases": [
```

Comment:
what does aliases mean?

Reply:
It just states the aliases of YAML language.